server/src/index.mjs

```diff
@@ -82,16 +82,16 @@ server.on('upgrade', function upgrade(request, socket, head) {
 });
 
 // Starter user(s)
-logger.info('Adding user 123...'); stateManagement.addUser('123'); userManagement.addUser('123');
-logger.info('Adding user 456...'); stateManagement.addUser('456'); userManagement.addUser('456');
-logger.info('Adding user 789...'); stateManagement.addUser('789'); userManagement.addUser('789');
+stateManagement.addUser('123'); userManagement.addUser('123');
```

Comment:
Can you remove these three lines? With the appId changes adding 123, 456, 789 is preventing 123~A, 456~A, and 789~A from being added which are commonly used userIds.